Gemfile

```diff
@@ -5,5 +5,6 @@ gem "mocha"
 gem "rake"
 gem "simplecov"
 gem "codeclimate-test-reporter", "1.0.0.pre.rc2"
+gem "pry"
```

Comment:
Personal opinion: pry tends to sneak into dependencies for temporary debugging, but not actually be used long-term, so keeping it committed as a dev dependency doesn't feel appropriate.